kartograf/coverage.py

```diff
@@ -11,28 +11,34 @@ def coverage(map_file, ip_list_file):
     rpki_masks = []
     for line in map_file:
         pfx, _ = line.split()
-        ipn = ipaddress.ip_network(pfx)
+        try:
+            ipn = ipaddress.ip_network(pfx)
+        except ValueError:
+            print(f"Invalid IP network provided: {pfx}")
+            continue
         netw = int(ipn.network_address)
         mask = int(ipn.netmask)
         rpki_masks.append(mask)
         rpki_nets.append(netw)
 
     net_masks = np.array(rpki_masks)
     network_addresses = np.array(rpki_nets)
+    zipped = list(zip(net_masks, network_addresses))
 
     addrs = []
     for line in ip_list_file:
-        ip = int(ipaddress.ip_address(line.rstrip('\n')))
-        addrs.append(ip)
+        try:
+            ip = ipaddress.ip_address(line.rstrip('\n'))
+            addrs.append(int(ip))
+        except ValueError:
+            continue
```

Comment:
printing the invalid IPs here can get quite noisy if the user fetched those IPs from their node and there are onion or i2p addresses, so I opted to skip silently. 
alternatively, we could keep a tally of invalids (optionally by type).

Reply:
I would prefer it if we fail on the first invalid and tell the user the content of the offending line. This is easier and better than continuing and showing a result at the end that may not actually contain what the user is expecting because they provided the wrong data without knowing. If they provided onion addresses they must have misunderstood something and I think it's reasonable that they filter their list themselves. But they could also simply have a formatting issue in their IPv6s or something and then the result could show plausible but incorrect data.